example/lava-docker.job

```diff
@@ -0,0 +1,53 @@
+# Zephyr JOB definition for frdm-k64f
+device_type: 'frdm-k64f'
+job_name: 'zephyr docker'
+
+timeouts:
+  job:
+    minutes: 3
+  action:
+    minutes: 3
+  actions:
+    wait-usb-device:
+      seconds: 40
+
+priority: medium
+visibility: public
+
+actions:
+
+- test:
+    docker:
+        image: kumargala/ci-run
+        wait:
+          device: false
+
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: kumar
+        run:
+          steps:
+          - ls -lsR /dev
+          - export
+          - pyocd list
+          - lava-test-case test5 --result pass
+          - lava-test-case test6 --result fail
+          - echo "<LAVA_SIGNAL_STARTTC ztest-1>"
+          - echo "foobar"
+          - echo "<LAVA_SIGNAL_ENDTC ztest-1>"
+          - echo "<LAVA_SIGNAL_TESTCASE TEST_CASE_ID=ztest-1 RESULT=safskip>"
+          - wget -q https://people.linaro.org/~kumar.gala/zephyr-sanitycheck-hw.tar.gz
+          - tar xf zephyr-sanitycheck-hw.tar.gz
+          - cd zephyr
+          - west init -l
+          - west config zephyr.base zephyr
+          - ./scripts/sanitycheck -i -p frdm_k64f --device-testing --device-serial /dev/ttyACM0 --test-only --west-flash
```

Comment:
Assumption here about the serial device being /dev/ttyACM0.  Need to go from something like ANDROID_SERIAL to the device.  Here's a quick python script w/hardcoded serial number.

```
import pyudev

context = pyudev.Context()

board_id = "02400b0131754e45001c0019948500046461000097969900"

for device in context.list_devices(subsystem='tty', ):
    if device.properties.get("ID_SERIAL_SHORT"):
        if board_id == device.properties["ID_SERIAL_SHORT"]:
            print(device.device_node)
```